CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ Pearl changelog
 
 ## [Unreleased]
 #### Added
+- B+ Tree indexes (#84)
```

Comment:
Please, add a line about Range Indices here.

Additionally: it is better to write a link to the Pull Request in braces.